### PR TITLE
Internationalization and Localization testing

### PR DESCRIPTION
**Pull request by AyyalAwwad**
Internationalization and Localization testing methods ensure that the localized software meets a local user’s expectations in terms of language, features, and user experience.

### CODE REVIEW

File: Paintroid/build.gradle [hunk 1]
```diff
@@ -7,6 +7,7 @@ android {
     compileSdkVersion 19
     buildToolsVersion "19.1.0"
 
+
```
Comment (X4mp) [inline Paintroid/build.gradle]: please no empty newlines

File: Paintroid/build.gradle [hunk 2]
```diff
@@ -15,6 +16,11 @@ android {
         testInstrumentationRunner "pl.polidea.instrumentation.PolideaInstrumentationTestRunner"
 		versionCode 1
 		versionName "1.0"
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
```
Comment (X4mp) [inline Paintroid/build.gradle]: I'm not sure if 2 testIntrumentationRunner work with Jenkins

**Comment by X4mp** [comment]
You are missing a rebase  to the current develop branch. With your branch the crop tool to resize refactoring which is already in the develop should not come up as commited changes.

File: Paintroid/build.gradle (hunk 1 above)
Comment (AyyalAwwad) [inline Paintroid/build.gradle]: Ok, I have removed the empty newline form gradle

File: Paintroid/build.gradle (hunk 2 above)
Comment (AyyalAwwad) [inline Paintroid/build.gradle]: The instrumentation runner is used for Espresso setup purposes

**Pull request closed without merging**